src/app/tree/tree-content.demo.ts

```diff
@@ -14,7 +14,7 @@ import { SohoTreeComponent } from 'ids-enterprise-ng';
 })
 export class TreeContentDemoComponent {
 
-  @ViewChild(SohoTreeComponent) tree: SohoTreeComponent;
+  @ViewChild(SohoTreeComponent, { static: true }) tree: SohoTreeComponent;
```

Comment:
How do we know whether or not to use the new `static` seems like its a mixture in this PR of false and true?

Reply:
@tmcconechy that's true, and it's a new feature from angular. What it means is this:
- static: true - hookup a ViewChild/Children before change detection runs - this happens once and no future change detection will update the viewChild.
- static: false - hookup the normal way where change detection can change your ViewChild or ViewChildren. 

Any case that angular did automatically update seems legit to me. They seem to inspect the dom for things requiring change detection to run for a viewChild to be set. If so they use static: false, else static: true. In most cases ng update changes these correctly, but when it cannot decide you'll get a message with the things you need to resolve.

Reply:
Cool if its automatically resolving this that seems legit!

Reply:
So the basic rule would be:
use static: false if you want a view child for an element inside and ngIf or ngSwitch. 
use static: true if the element is in the template before change detection and never needs to be updated after initial hookup.

Reply:
You must use the new option. So if you are unsure then make it static: false and it will behave as before.